src/normal_form_game.jl

```diff
@@ -53,6 +53,61 @@ function Base.show(io::IO, player::Player)
     Base.print_array(io, player.payoff_array)
 end
 
+# delete_action
+
+"""
+    delete_action(player, action, player_idx)
+
+Return a new Player instance with the action(s) specified by `action`
+deleted from the action set of the player specified by `player_idx`.
+
+# Arguments
+
+- `player::Player` : Player instance.
+- `action::Union{PureAction,AbstractVector{<:PureAction}}`: The action(s) to be
+  deleted.
+- `player_idx::Integer` : Index of the player to delete action(s) for.
+
+# Returns
+
+- `::Player` : `Player` instance with the action(s) deleted as specified.
+
+# Example
+
+julia> player = Player([3 0; 0 3; 1 1])
+julia> player.payoff_array
+3×2 Array{Int64,2}:
+ 3  0
+ 0  3
+ 1  0
+julia> player_delete1 = delete_action(player, 3, 1)
+julia> player_delete1.payoff_array
+2×2 Array{Int64,2}:
+ 3  0
+ 0  3
+julia> player_delete2 = delete_action(player, 1, 2)
+julia> player_delete2.payoff_array
+3×1 Array{Int64,2}:
+ 0
+ 3
+ 1
+
+"""
```

Comment:
Either

* remove the 4 lines between `"""` and `function delete_action(...)` (and move the link to stackoverflow to somewhere else); or

* surround `""" ... """` with `@doc` and `delete_action`:

  ```jl
  @doc """

  ...

  """ delete_action
  ```

  (Search for `@doc` within this file for a similar case.)